.github/workflows/integration-tests-soak.yml

```diff
@@ -59,14 +79,11 @@ jobs:
     steps:
       - name: Checkout the repo
         uses: actions/checkout@v3.0.2
-      - name: Install Nix
-        uses: cachix/install-nix-action@d64e0553100205688c0fb2fa16edb0fc8663c590 # v17
-        with:
-          nix_path: nixpkgs=channel:nixos-unstable
       - name: Run Tests
-        uses: smartcontractkit/chainlink-github-actions/chainlink-testing-framework/run-tests@cd9e407a6e0aa8ef7f315067644c0333846ffd40 # v2.0.24
+        uses: smartcontractkit/chainlink-github-actions/chainlink-testing-framework/run-tests@13437f3be1c82243a8c44eabc043a4fa58d72d73 # v2.0.20
```

Comment:
wondering: why is this going from v2.0.20 => v2.0.24

Reply:
Bumped accordingly to latest